locale.html

```diff
@@ -90,6 +90,55 @@ <h1><ins>TimeZonesOfLocale ( _loc_ )</ins></h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-direction-of-locale" aoid=DirectionOfLocale>
```

Comment:
Editorial suggestion: use `<ins class="block">` to wrap the whole `<emu-clause>` instead of all the inline `<ins>` tags.

Reply:
> Editorial suggestion: use `<ins class="block">` to wrap the whole `<emu-clause>` instead of all the inline `<ins>` tags.

Let me deal with that separately